src/resources/postcss/components/_small-cta.pcss

```diff
@@ -0,0 +1,31 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Small CTA (Tooltips - ET, ET+, RSVP)
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.tribe-events-c-small-cta-link {
+		@mixin desktop-body-3-bold;
+
+		border-bottom: 2px solid #ddd;
```

Comment:
Oh just spotted this, assuming we want to stick to our hopefully consistently designed colors and stick to vars?

Reply:
Good catch! missed that one.